subjects/sumascii/README.md

```diff
@@ -0,0 +1,18 @@
+## sumascii
+
+### Instructions
+
+Create a program that receives an argument and displays the sum of the ASCII value of each character. The sum will be a number representing the value after the sum of the bytes. If there is an invalid input print 0.
+
+### Usage
+
+```console
+$ go run . "hi" | cat -e
```

Comment:
Another thing the sol doesnt cover is for example this case
```
go run . "hid" | cat -e
53$
```
it was supposed to be 309 and not 53 right?

Reply:
Wouldn't it be right, since we are adding bytes and not integers as written in the description? A byte in Go is an uint8, so it has a size of 8 bits. This means the numbers it can represent go from 0 to 255. 255 is ```1111 1111```, when you add 1, you get ```1 0000 0000```, and because it can only allocate 8 bits in memory, it will drop the leftmost bit, leaving ```0000 0000```.

This means that when the result is over 255 it will start counting from 0, for example, 256 will be 0, and 257, 1. That's why the solution in my humble opinion, is 53. Please let me know if I'm wrong and I'll correct it.

Reply:
Yes you're right, my bad i was looking at it as integers